### DISCUSSION TITLE
REQ Style Selector - JSON selector

### DISCUSSION THREAD

**Discussion by barepixels** [lllyasviel/Fooocus#2692] [Ideas]
I just found a useful extension for A1111 call [StyleSelectorXL_JsonSelector](https://github.com/benjamin-bertram/StyleSelectorXL_JsonSelector)

I wish Fooocus can do the same.  Able to swap style.json.  This way we don't have to deal with a very long list.

**Comment by wisdombox44** [comment]
That is Really Nice to select or disable some styles, 
Generate All Styles In Order is great also
but does not have prewievs, and can not mix multiple styles.
It is fully compitable with fooocus styles, 
Please Developers make Style Selector some times we only need few styles and we get lost in 250+ styles
